core/storage/statistics/gae_models.py

```diff
@@ -875,6 +875,24 @@ def save_multi(cls, exploration_stats_dicts):
             exploration_stats_models.append(stats_instance)
         cls.put_multi(exploration_stats_models)
 
+    def to_dict(self):
+        """Creates a dict representation of the ExplorationStatsModel.
+
+        Returns:
+            dict. A dictionary representation on the ExplorationStatsModel.
+            Each key is an attribute of the model.
+        """
+        return {'exp_id': self.exp_id,
```

Comment:
As a note for the future, our convention for dicts in this codebase is to break after the opening brace, viz.:

```
        return {
            'exp_id': self.exp_id,
            ...,
        }
```

Reply:
This is removed, appreciate the note.